etl/src/main/java/org/pih/hivmigration/etl/sql/Setup.groovy

```diff
@@ -16,6 +16,12 @@ class Setup extends SqlMigrator {
             DELIMITER ;
         ''')
 
+        executeMysql('''
+            DROP FUNCTION IF EXISTS is_number;
+            CREATE FUNCTION is_number (_value text) RETURNS boolean DETERMINISTIC
+            RETURN concat('', _value * 1) = _value;
```

Comment:
this is an interesting one! might want to add a link to the source for this in case anyone is doubting your logic, this one seemed a bit cryptic to me, but I googled and found the link I'm guessing you did:  

https://stackoverflow.com/questions/5064977/detect-if-value-is-number-in-mysql

:)

Reply:
That's the one, yeah, the comment is a good idea